driver/src/main/scala/org/mongodb/scala/Observable.scala

```diff
@@ -39,14 +42,34 @@ object Observable {
 }
 
 /**
- * A `Observable` represents a MongoDB operation.
+ * A `Observable` represents a MongoDB operation and implements the `Publisher` interface.
```

Comment:
This file contains the big change. Previously we has the async `Observable` and a scala implicit `Observable` that would add all these helpers.

Now they are part of the trait and the `Observable` itself implements the `Publisher` interface.